Xamarin.Forms.Platform.iOS/CollectionView/CollectionViewController.cs

```diff
@@ -151,11 +151,11 @@ UICollectionViewCell GetPrototype()
 
 		void RegisterCells()
 		{
-			CollectionView.RegisterClassForCell(typeof(DefaultHorizontalListCell), DefaultHorizontalListCell.ReuseId);
-			CollectionView.RegisterClassForCell(typeof(DefaultVerticalListCell), DefaultVerticalListCell.ReuseId);
-			CollectionView.RegisterClassForCell(typeof(TemplatedHorizontalListCell),
-				TemplatedHorizontalListCell.ReuseId);
-			CollectionView.RegisterClassForCell(typeof(TemplatedVerticalListCell), TemplatedVerticalListCell.ReuseId);
+			CollectionView.RegisterClassForCell(typeof(HorizontalDefaultCell), HorizontalDefaultCell.ReuseId);
+			CollectionView.RegisterClassForCell(typeof(VerticalDefaultCell), VerticalDefaultCell.ReuseId);
+			CollectionView.RegisterClassForCell(typeof(HorizontalTemplatedCell),
+				HorizontalTemplatedCell.ReuseId);
+			CollectionView.RegisterClassForCell(typeof(VerticalTemplatedCell), VerticalTemplatedCell.ReuseId);
```

Comment:
We should make sure that the ReuseId are unique and match the type name for the first release.